about.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>FABL:TEMPLATE</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="/css/styles.css" />
+    <link href="https://fonts.googleapis.com/css?family=Asar|Sansita" rel="styles.css">
+    <script src="main.js"></script>
+</head>
+    <body>
+        <header>
+            <img id="logo" src="images/logos/logo-FABL-horz_07.png">
+        <nav>
+        <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">RubyRoute</a></li>
+            <li><a href="#">Products</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        </nav>
+        </header>
+        <div>
+            <section id="">
+                <div id="">
+                <article>
+                        <h1>About FABL</h1>
+                        <h3>Who are we?</h3>
+                            <p>FABL is a company made by parents, for parents. With the safety of children in mind, our company works day in and day out to be sure that parents have something to help them keep their children safe and sound
+                                                in any situation that may concern them normally. With our products, parents can be sure that they know where their child is when traveling alone, check up on their personal health, keep busy schedules simple and clean, and be sure their child is transported safely to their destination along with games and entertainment for children to keep them happy and engaged. Nothing is more important to us than the relationship between parent and child, therefore we will continue to work our hardest to ensure that parents and children alike will have a seamless, entertaining, and fun-filled experience with our applications.</p>
+                </article>
+                </div>
+                <div id="">
+                <h3>Team Members</h3>
```

Comment:
Again with the headers - go with <h2> for the team members header and <h3>s for the rest.

Reply:
I think your comments used whatever header you told me to use for your comment so i can't see which you told me to use